ir/state.cpp

```diff
@@ -156,7 +157,7 @@ void State::addJump(const BasicBlock &dst0, expr &&cond) {
   auto &data = predecessor_data[dst][current_bb];
   data.second.add(memory, cond);
   data.first.UB.add(domain.UB(), cond);
-  data.first.path.add(move(cond));
+  data.first.path.add(move(cond) && domain.noreturn());
```

Comment:
This shouldn't be needed here. Path only tracks the paths constraints, i.e., the branching conditions